src/app/components/BibPage/components/DefinitionNoteField.jsx

```diff
@@ -1,21 +1,18 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { useBibParallel } from '../../../context/Bib.Provider';
-
-const DefinitionNoteField = ({ value }) => {
-  const { parallel } = useBibParallel('note');
-
-  const list = parallel ?? [value];
 
+const DefinitionNoteField = ({ values }) => {
+  // type value = Note[]
   return (
     <ul>
-      {list
+      {values
         .map((note, idx) => {
-          // Check by object reference
-          return note[1] === value ? (
+          return note && note.prefLabel ? (
             <>
-              {note[0] ? <li key={(idx - 1).toString()}>{note[0]}</li> : null}
-              <li key={idx.toString()}>{note[1].prefLabel ?? note[1]}</li>
+              {note.parallel ? (
+                <li key={`${note.noteType}_${idx}`}>{note.parallel}</li>
```

Comment:
Seems like this should be unique, but any reason the key isn't just something like ```{`${idx}-parallel`}```?

Reply:
``{`${note.noteType}_${idx}`}`` seems to be a better reflection that the list being rendered is related to Notes, its index and the subject. There can be additional Notes in a different loop which would then have a similar key if we used your suggestion. For example, we may have a list with subjects of "Biography" and "Language" which could have two notes in their list. By using ``{`${idx}-parallel`}`` both the subjects keys would be the same and React would yell at us. 

Additionally, there are list items being rendered in a similar component (`DefinitionField`) outside of this `DefinitionNoteField` which would potentially have the same key structure that you point out above. Appending `parallel` to an `idx` potentially may set duplicate keys. 

However, if you'd prefer, it can be changed. I argue against making the change.